lib/utils.js

```diff
@@ -42,6 +42,11 @@ function uriBase(req) {
     return uriAbs(req) + (req.baseUrl || '');
 }
 
+function uriRelative(uri) {
+    var relative = path.basename(uri);
```

Comment:
I would not rely too much on this function. This function just gives the latest part in a path `hello/nicola/folder` will return `folder`. The idea of relative is that a link is relative to a particular directory. Although this solves the problem for the specific case, this function needs a second parameter that is the base from which we want to find the path of `uri` relative to the `base`